BlueprintUICommonControls/Sources/Box.swift

```diff
@@ -102,6 +102,7 @@ extension Box {
 
     public enum CornerStyle {
         case square
+        case semicircular
```

Comment:
I wonder if “pill” might be clearer? No preference, but in my head that makes me more immediately realize what it does.

Reply:
I'm open to changing it of course, but biased towards `semicircular` because

- it fits grammatically with the other cases (as an adjective)
- it is a  generally understood geometric term (like square)

In my mind, `pill` is jargon and no different than `chip`, what Google's Material calls it. If you're familiar with the argot, it's familiar. Since most people working with Blueprint share this familiarity, I think it's a fine choice.

Reply:
Way overthought bikeshed:

I think part of the issue here is that the existing cases are purely the corner style where this new case is wanting to describe a shape as a whole (that is, affecting more than each corner individually). And I think we're left trying avoid using a “shape” word to describe a “corner.”

And my mind _still_ reads it as a shape and expects a half a circle that's flat on the other side (that is to say, a literal semicircle).

FWIW SwiftUI avoids this issue by having this as a shape type (called [Capsule](https://developer.apple.com/documentation/swiftui/capsule)) rather than a corner style.

I could see a few options:

1. Rename this type/property to be “Shape” and then something like “Capsule” or “Pill” seems more right to me
2. Put this into its own element type altogether (again, as Pill or Capsule IMO)
3. Land as-is with the same or a different name (in which case I'd probably still lean against “semicircle” and just accept conflating shape and corner a bit)

(And Google's wrong, “chip” is a weird name)

Reply:
capsule is also a good name for this property 

I wouldn't make this a different element since we'd have to duplicate all the shadow and border style stuff.

Reply:
Given the enthusiasm for `capsule `, I updated it. Still, I think it stands in contrast to its sibling cases.

> And my mind still reads it as a shape and expects a half a circle that's flat on the other side (that is to say, a literal semicircle).

That's fair, though this is one impression I tried to avoid when renaming it from `semicircle` to `semicircular`. The latter may be ascribed to an object forming a semicircle, which is a fair attribution to such corners. This is similar if not identical semantically to `square`, I'd argue.

Reply:
Yeah, we've definitely got a less than idea parallelism here. I think were this has landed is more intuitive, though less … precise.